.github/workflows/bump-cli.yml

```
@@ -0,0 +1,38 @@
+name: Bump CLI version
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/actions/create-pr/action.yml
+  schedule:
+  - cron: 0 0 */14 * * # run every 14 days
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Bump CLI
+        env:
+          GH_TOKEN: ${{ github.token }}
```

I might be mistaken, but I think this is what you need to do:

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

The priority level for `gh` cli is `GH_TOKEN` then `GITHUB_TOKEN` I think, but both work; can change it if you'd like 😄 

```sh
> gh help environment
GH_TOKEN, GITHUB_TOKEN (in order of precedence): an authentication token for github.com
API requests. Setting this avoids being prompted to authenticate and takes precedence over
previously stored credentials.
...
```

---

Thanks...TIL.